redfish/data_source_redfish_dell_idrac_attributes_test.go

```diff
@@ -0,0 +1,37 @@
+package redfish
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccRedfishiDRACDataSource_fetch(t *testing.T) {
```

Comment:
Is it possible to include negative test ?

Reply:
There are no filters(attributes to specify) available for this datasource so, can't add any negative test case.